package.json

```diff
@@ -7,13 +7,14 @@
   },
   "dependencies": {
     "ajv": "6.10.2",
-    "logzio-node-debug": "^2.0.0",
+    "logzio-node-debug": "2.0.0",
     "ramda": "0.26.1",
     "valid-url": "1.0.9"
   },
   "devDependencies": {
     "chai": "4.2.0",
     "chai-as-promised": "7.1.1",
```

Comment:
are all of these libs used? I think that `chai-as-promised` is not necessary, right<